docker/package/defaults/tezos-baking.conf

```diff
@@ -0,0 +1,9 @@
+# vim: ft=sh
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# shellcheck disable=SC2034
+DATA_DIR="/var/lib/tezos/.tezos-client"
+NODE_RPC_ENDPOINT="http://localhost:8732"
```

Comment:
Since this is common to `baking-<proto>` services of different protocols it is somewhat problematic when running multiple baking instances for different ones.

The issue is not introduced here and the same problem is true for running single `node`/`baker`/`endorser` services, so I suggest merging this PR and solving it somewhere else.
It's not really a breaking problem, since one can change the configurations, but it would be nice to have default values that can support multiple protocols by using different ports.

A similar thing can be said for `tezos-client` that will just "aim" at this port. For that I suggest extending the scope of #148 to also add `tezos-client-mainnet` `tezos-client-edo2net` and `tezos-client-florencenet` aliases to avoid the need to manually set up the RPC port.